app/src/Components/Base/Content.js

```diff
@@ -8,14 +8,51 @@ import DataSource from './../DataSource/DataSource';
 import NotFoundComponent from '../Error/NotFoundComponent'
 import Login from '../Login/Login';
 
+const PrivateRoute = ({ component: Component, ...rest }) => (
+    <Route {...rest} render={(props) => (
+        isAuthentificated(rest)
+            ? <Component {...props} />
+            : <Redirect to={{
+            pathname: '/login',
+            state: { from: props.location }
+        }} />
+    )} />
+);
+
+function isAuthentificated(props){
+    let cookieValue = getCookieValue('token');
+    if(cookieValue !== null && cookieValue.length > 5){
```

Comment:
Longer than 5 seems like an arbitrary check. 
Either check the token's validity otherwise without its length, or just use a library to parse it properly.

Reply:
Fixed in 8ba6499f998781a8ec688f69da00d68edc66de6d.